module/VuFind/src/VuFind/ChannelProvider/Facets.php

```diff
@@ -72,31 +74,20 @@ class Facets extends AbstractChannelProvider implements TranslatorAwareInterface
      */
     protected $maxValuesToSuggestPerField;
 
-    /**
-     * Search results manager.
```

Comment:
Maybe we should change the PHPdoc on the constructor to read "Search results manager" so this slightly more complete documentation is not lost in the refactoring.